docs/docsite/rst/developing_modules_general_aci.rst

```diff
@@ -0,0 +1,1851 @@
+
+Now that we have forked our repository, let's clone the forked repository on our local machine.
+
+**Clone**  
+  Clone allows you to copy a repository to your local machine.
+
+* Clone the forked repo by going to the terminal and enter the following command:
+
+.. code-block:: Blocks
+
+  git clone https://github.com/<Forked Organization>/ansible-aci.git
+
+**Naming Convention**
+  "origin" is the default name for the first Git remote of a cloned repository. In this case, it represents your forked repo where you are going to make changes, commit, and push your code to GitHub.
+
+* Verify the name of the Git remote of your forked repository by going to the terminal and enter the following command:
+
+.. code-block:: Blocks
+
+  git remote -v
+
+You should see in the output your repository listed after the name origin.
+.. code-block:: Blocks
+
+origin        https://github.com/<Forked Organization>/ansible-aci.git (fetch)
+origin        https://github.com/<Forked Organization>/ansible-aci.git (push)
+
+To be able to retrieve the latest changes made to the upstream project repo (CiscoDevNet/ansible-aci), we need to add it as a second Git remote. We recommend calling this second remote "upstream" and we will keep referring to it as upstream in the rest of the document.
+
+* Add the upstream repo as a new Git remote:
+
+.. code-block:: Blocks
+
+  git remote add upstream https://github.com/CiscoDevNet/ansible-aci.git
+
+Adding the main repository "upstream" is a one-time operation.
+Now that we have added the upstream repo as a remote, we can make sure that our local master branch is up-to-date with the upstream repository.
+
+* Update the local master branch from the upstream repository:
+
+.. code-block:: Blocks
+
+  git checkout master
+  git pull upstream master
+
+Now that our local master branch is up-to-date with the upstream repo, we can create a feature branch.
+
+**Branch**
+  Creating branches makes it easier to fix bugs, add new features, and integrate new versions after they have been tested in isolation. Master is the default branch of the local repository. Each time you need to make changes to a module or create a new module, we recommend that you create a new dedicated branch from master.
+
+* Create a branch from master by using the following commands on the terminal:
+
+.. code-block:: Blocks
+
+  git checkout master
+  git checkout -b <new-branch-name>
+  git branch
+
+You now have a clean branch of the latest master, where you can make all of your changes. By keeping your changes in a dedicated branch, you can keep the master branch clean and on track with the upstream master. This makes it easier to keep the local master branch updated without needing to merge code or rebase the master branch. As a best practice, we recommend that you do not commit changes to your local master branch but commit them to a dedicated feature branch.
```

Comment:
Do we want to write in the you style? I recall we did decided before this would be to informal for our docs